src/apps/loyalty/server/route_handlers.tsx

```diff
@@ -60,31 +60,31 @@ export function Inquiries(req: Request, res: Response, next: NextFunction) {
   if (info.confirmed_buyer_at) {
     markCollectorAsLoyaltyApplicant(CURRENT_USER.accessToken)
       .then(profile => {
-        return res.redirect(req.baseUrl + "/thank-you")
+        res.redirect(req.baseUrl + "/thank-you")
       })
```

Comment:
if this return does nothing, this line could turn into `.then(profile => res.redirect(req.baseUrl + "/thank-you") )`